pages/common/audacious.md

```diff
@@ -0,0 +1,32 @@
+# audacious
+
+> An open-source audio player.
+> More information: <https://audacious-media-player.org/>.
+
+- Launch the program:
+
+`audacious`
+
+- Enqueue a specific directory of audio files:
+
+`audacious --enqueue {{path/to/directory}}`
+
+- Start/stop playback:
```

Comment:
```suggestion
- Start or stop playback:
```

Optional fix: `or` may be a better choice where delimited terms don't contain `or` themselves.